clients/admin-ui/src/features/system/DescribeSystemStep.tsx

```diff
@@ -47,12 +46,14 @@ const SystemHeading = ({ system }: { system?: System }) => {
 
 interface Props {
   onSuccess: (system: System) => void;
+  onCancel: () => void;
```

Comment:
these steps were broken in `main`: hitting the cancel/back button didn't do anything when trying to edit a system. this was because this component used to be used in two places
1. Config wizard
2. Editing a system `/system/configure`

It seems like this component (and the other relevant steps, `PrivacyDeclarationStep`, `ReviewStep`), were only changed to work with the config wizard and not editing a system (hence the import from the config wizard slice).